### PR TITLE
Add patient to notification MUZIMA-359

### PR DESCRIPTION
**Pull request by sthaiya**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/muzimaconsultation/handler/ConsultationQueueDataHandler.java [hunk 1]
```diff
@@ -21,19 +21,7 @@
...
-import org.openmrs.Location;
-import org.openmrs.Obs;
-import org.openmrs.Patient;
-import org.openmrs.PatientIdentifier;
-import org.openmrs.PatientIdentifierType;
-import org.openmrs.Person;
-import org.openmrs.PersonName;
-import org.openmrs.Role;
-import org.openmrs.User;
+import org.openmrs.*;
```
Comment (nribeka) [inline api/src/main/java/org/openmrs/module/muzimaconsultation/handler/ConsultationQueueDataHandler.java]: @sthaiya do a follow up commit to fix this :)

File: api/src/main/java/org/openmrs/module/muzimaconsultation/handler/ConsultationQueueDataHandler.java [hunk 2]
```diff
@@ -58,11 +46,7 @@
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```
Comment (nribeka) [inline api/src/main/java/org/openmrs/module/muzimaconsultation/handler/ConsultationQueueDataHandler.java]: And this one.

File: omod/src/main/java/org/openmrs/module/muzimaconsultation/web/controller/PatientController.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * TODO: Write brief description about the class here.
+ */
+@Controller
+public class PatientController {
+
+    @RequestMapping(value = "/module/muzimaconsultation/patients.json", method = RequestMethod.GET)
+    @ResponseBody
+    public List<Object> getAllPatients(final @RequestParam(value = "param", required = false) String param) {
+        if (StringUtils.isEmpty(param) || param.equalsIgnoreCase("undefined")) {
```
Comment (nribeka) [inline omod/src/main/java/org/openmrs/module/muzimaconsultation/web/controller/PatientController.java:42]: @sthaiya do a follow up commit to wrap all this operation inside if (Context.isAuthenticated()) to make sure that this part can't be called without authenticating :)

Comment (sthaiya) [inline omod/src/main/java/org/openmrs/module/muzimaconsultation/web/controller/PatientController.java:42]: Follow up commit submitted

**Pull request merged**